libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/theme/components/Button.kt

```diff
@@ -58,6 +58,9 @@ import io.element.android.libraries.designsystem.preview.PreviewGroup
 
 // Designs: https://www.figma.com/file/G1xy0HDZKJf5TCRFmKb5d5/Compound-Android-Components?type=design&mode=design&t=U03tOFZz5FSLVUMa-1
 
+// Horizontal padding for button with low padding
+internal val lowPaddingValue = 4.dp
```

Comment:
Should be name lowHorizontalPaddingValue then?

Reply:
212655610595afff99af2bfd98feeb57270629ab